src/Cat.java

```diff
@@ -0,0 +1,31 @@
+public class Cat extends Animal {
+    private static final int RUNNING_RESTRICTION = 200;
```

Comment:
it is not a constant. According to my knowledge, different cats can run different distance